### ISSUE TITLE
Waypoints mixed up or not removed after cache owner updated/removed it

### ISSUE THREAD

**Issue by knoepfchendruecker** [cgeo/cgeo#3650]
GC4XK6D recently introduced another trailhead waypoint.

before: RECHT1 was the only trailhead
after: RECHT1 and RECHT2 do exist as trailhead waypoints.

Updating the cache in cgeo now displays the description for RECHT2 in RECHT1 and the description of the final-waypoint in the description of RECHT2.

![2014-03-02 22 48 08](https://f.cloud.github.com/assets/6894621/2366596/451608f4-a711-11e3-853a-ad07782f68b4.png)

**Comment by Lineflyer** [comment]
That should be solved implicitly with the fix fpr #4686

**Comment by Lineflyer** [comment]
According to a user feedback on support this problem seems still to be existing:
The owner of a cache modified it and afterwards updated his listing in c:geo but now has some mixup of waypoints and texts and a wrong sort order.

I requested a screenshot in order to compare with the current website waypoints.

**Comment by bau-ch** [comment]
the same here: It's not possible to delete own waypoints in c:geo (2016.05.09) with CM13.0 on a Moto G3

**Comment by Lineflyer** [comment]
There is a new user report with some an issue.
I will assign myself to test this.

**Comment by Lineflyer** [comment]
The problem is now slightly different. Will open a new issue.
